tests/test_spark.py

```diff
@@ -389,3 +389,67 @@ def test_task_stdout():
         assert stdout_file["size"] > 0, "stdout file should have content"
     finally:
         sdk_install.uninstall(utils.SPARK_PACKAGE_NAME, service_name)
+
+
+@pytest.mark.sanity
+def test_supervise_conflict_frameworkid():
```

Comment:
overall, the test code looks good but teamcity CI fails because it looks you're trying to submit a spark job after the dispatcher has already been uninstalled [in the previous test](https://github.com/mesosphere/spark-build/pull/506/files#diff-2caedadc3f4e69445462b60b70423fcfR391). Suggest either reusing a previous dispatcher (saves time) or redeploying one for your test

Reply:
As far as I understood, in this module dispatcher is auto installed, it will be reused. The teamcity CI fails because of [this](https://teamcity.mesosphere.io/viewLog.html?tab=buildLog&logTab=tree&filter=debug&expand=all&buildId=1800891&_focus=19929) reason.

Reply:
I saw [this error](https://teamcity.mesosphere.io/viewLog.html?tab=buildLog&logTab=tree&filter=debug&expand=all&buildId=1800891&_focus=18660) which caused the diag bundle collection due to failure.

Reply:
@farhan5900, it seems that something is odd with the way the fixture works as pointed out by @samvantran. I'd suggest to do some debugging of the test itself and look at the cluster state at the moment of calling `utils.submit_job` by pausing the execution.